.github/workflows/rust.yaml

```diff
@@ -15,7 +15,7 @@ env:
   CARGO_INCREMENTAL: 0
   CARGO_TERM_COLOR: always
   # Build smaller artifacts to avoid running out of space in CI
-  RUSTFLAGS: -C strip=debuginfo
+  RUSTFLAGS: -C strip=debuginfo -C opt-level=s
```

Comment:
Should we add todo to remove this once we will have dedicated runners? I believe @Genysys wanted to work on that as he would have free time

Reply:
It might be still useful for dedicated runners, so not 100% sure we need a TODO here

Reply:
I think it might be good to have nice stacktrace for tests and faster build times in general

Reply:
Linking of stripped libraries should actually be faster I think

Reply:
Yeah, right, but optimization is not :)